src/main/java/com/epam/pages/popup/SubjectPopup.java

```diff
@@ -1,6 +1,127 @@
 package com.epam.pages.popup;
 
+import com.epam.helpers.SharedTestData;
 import com.epam.pages.common.CommonPopup;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.List;
 
 public class SubjectPopup extends CommonPopup {
+
+    @FindBy(id = "name")
+    private WebElement nameField;
+    @FindBy(id = "teacher")
+    private WebElement teacherSelect;
+    @FindBy(xpath = "//*[@class='selection']/span")
+    private WebElement selectedTeachers;
+    @FindBy(className = "select2-search__field")
+    private WebElement teacherSelectField;
+    @FindBy(xpath = "//*[@id='select2-teacher-container']/li")
+    private List<WebElement> listOfSelectedTeachers;
+    @FindBy(xpath = "//*[@id='select2-teacher-container']/li/button")
+    private List<WebElement> xButtonsOfSelectedTeachers;
+    @FindBy(className = "select2-selection__clear")
+    private WebElement xButtonOfTheTeacherList;
```

Comment:
Same as in the previous, please rename variable to xButtonOfTheTeacherSelectBox

Reply:
Resolved